task/05-regex-tasks.js

```diff
@@ -91,7 +93,7 @@ function getRegexForIPv4() {
  * @return {RegExp}
  */
 function getRegexForSSN() {
-   throw new Error('Not implemented');
+   return /^[0-9]{2}[1-9]{1}-[0-9]{1}[1-9]{1}-[0-9]{3}[1-9]{1}$/;
```

Comment:
Some special numbers are never allocated:
Numbers with all zeros in any digit group (000-##-####, ###-00-####, ###-##-0000).
**Numbers with 666 or 900-999 (Individual Taxpayer Identification Number) in the first digit group.**

not correct, highlighted rules are not validated